Userland/Libraries/LibCore/System.cpp

```diff
@@ -904,4 +904,10 @@ ErrorOr<void> mknod(StringView pathname, mode_t mode, dev_t dev)
 #endif
 }
 
+ErrorOr<void> mkfifo(StringView pathname, mode_t mode)
+{
+    TRY(mknod(pathname, mode | S_IFIFO, 0));
+    return {};
```

Comment:
```suggestion
    return mknod(pathname, mode | S_IFIFO, 0);
```

Reply:
Of course, not even sure what I was thinking here